unified-runtime/scripts/core/exp-graph.yml

```diff
@@ -0,0 +1,214 @@
+#
+# Copyright (C) 2025 Intel Corporation
+#
+# Part of the Unified-Runtime Project, under the Apache License v2.0 with LLVM Exceptions.
+# See LICENSE.TXT
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+#
+# See YaML.md for syntax definition
+#
+# TODO:
+# ZE_RESULT_ERROR_INVALID_GRAPH
+# ZE_RESULT_QUERY_TRUE
+# ZE_RESULT_QUERY_FALSE
+#
+--- #--------------------------------------------------------------------------
+type: header
+desc: "Intel $OneApi Unified Runtime Experimental APIs for Graph Record and Replay"
+--- #--------------------------------------------------------------------------
+type: handle
+desc: "Handle of record & replay graph object"
+class: $xGraph
+name: "$x_exp_graph_handle_t"
+--- #--------------------------------------------------------------------------
+type: handle
+desc: "Handle of record & replay executable graph object"
+class: $xGraph
+name: "$x_exp_executable_graph_handle_t"
+--- #--------------------------------------------------------------------------
+type: function
+desc: "Create a new record & replay graph instance explicitly."
+class: $xGraph
+name: CreateExp
+decl: static
+details:
+    - "Create a new record & replay graph instance explicitly."
+params:
+    - type: $x_context_handle_t
+      name: hContext
+      desc: "[in] Handle of the context object."
+    - type: $x_exp_graph_handle_t*
+      name: phGraph
+      desc: "[out] Pointer to the handle of the created graph object."
+    - type: void*
+      name: pNext
+      desc: "[out] pointer to extension-specific structure."
+returns:
+    - $X_RESULT_SUCCESS
+    - $X_RESULT_ERROR_INVALID_ARGUMENT
+--- #------------------------------------------k--------------------------------
+type: function
+desc: "Begin graph capture on the specified immediate command list."
+class: $xQueue
+name: BeginGraphCaptureExp
+params:
+  - type: $x_queue_handle_t
+    name: hQueue
+    desc: "[in] Handle of the queue on which to begin graph capture."
+  - type: void*
+    name: pNext
+    desc: "[in][optional] pointer to extension-specific structure."
+returns:
+    - $X_RESULT_SUCCESS
+    - $X_RESULT_ERROR_INVALID_ARGUMENT
+--- #--------------------------------------------------------------------------
+type: function
+desc: "Begin capturing commands into an existing graph on the specified immediate command list."
+class: $xQueue
+name: BeginCaptureIntoGraphExp
+params:
+  - type: $x_queue_handle_t
+    name: hQueue
+    desc: "[in] Handle of the queue on which to begin graph capture."
+  - type: $x_exp_graph_handle_t
+    name: hGraph
+    desc: "[in] Handle of the graph object to capture into."
+  - type: void*
+    name: pNext
+    desc: "[in][optional] pointer to extension-specific structure."
+returns:
+  - $X_RESULT_SUCCESS
+  - $X_RESULT_ERROR_INVALID_ARGUMENT
+--- #--------------------------------------------------------------------------
+type: function
+desc: "End graph capture on the specified immediate command list."
```

Comment:
Mention that this also returns a graph handle, and that if `BeginCaptureIntoGraphExp` was used then the output graph handle will be the same as provided there.